custom_components/easee/sensor.py

```diff
@@ -332,7 +354,11 @@ async def async_update(self):
             self._state = self.get_value_from_key(self._state_key)
             if self._state_func is not None:
                 charger_state = await self.charger.get_state(from_cache=True)
-                self._state = self._state_func(charger_state)
+                charger_config = await self.charger.get_config(from_cache=True)
+                try:
+                    self._state = self._state_func(charger_state)
+                except:
+                    self._state = self._state_func(charger_config)
```

Comment:
I think it is better to look at the _state_key and see if it contains "config" or "state"

Reply:
You're right, this was dirty from my side, getting late I guess :) My problem was that I needed it for the attribute keys as well, not only state_key. But later I changed the state to be config to, so I guess it works now.